### DISCUSSION TITLE
[Support] List of known quirks

### DISCUSSION THREAD

**Discussion by Unb0rn** [StarLabsLtd/firmware#230] [Q&A] [answered]
Hello! I'm not sure if this is the correct place to ask questions like these but I'll try
Is there a list of known quirks/workarounds for StarLabs hardware?
I'm running Linux on StarBook Mk VI and I have a couple of minor problems:

1. After upgrading the FW to 24.10 and finally getting SecureBoot (btw, thank you, guys) I wanted to migrate my LUKS setup to TPM. The problem is - all the PCRs are empty (except for PCR2). SecureBoot is active (booting UKI) and reboot changes nothing - zeroes in all PCRs. Am I doing something wrong?
2. S0ix aka "modern standby". Running Intel's S0ix self-testing tool I got that this state is unsupported. Is this related to ME being turned off? Any possible workarounds?
3. This generic simpledrm framebuffer thing. Good thing the issue is still open and it's possible to "just google" the problem, yet it would be nice to have some sort of doc like [this](https://support.starlabs.systems/articles/starbook-mk-v-intel-variant) but for newer hardware.

**Comment by Sean-StarLabs** [comment, marked as answer]
> Is there a list of known quirks/workarounds for StarLabs hardware?

Beyond https://github.com/StarLabsLtd/firmware/issues?


> After upgrading the FW to 24.10 and finally getting SecureBoot (btw, thank you, guys) I wanted to migrate my LUKS setup to TPM. The problem is - all the PCRs are empty (except for PCR2). SecureBoot is active (booting UKI) and reboot changes nothing - zeroes in all PCRs. Am I doing something wrong?

No, that's not supported yet - see #203 

> S0ix aka "modern standby". Running Intel's S0ix self-testing tool I got that this state is unsupported. Is this related to ME being turned off? 

No

>  Any possible workarounds?

No, we don't support S0ix (lacking a single compelling reason to do so).
